Content.Shared/Chemistry/Solution/Components/FitsInDispenserComponent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Shared.Chemistry.Solution.Components
+{
+    [RegisterComponent]
+    public class FitsInDispenserComponent : Component
+    {
```

Comment:
Would be nicer to have some doc comments in these classes. Examinable is pretty intuitive but "FitsInDispenser" isn't.

Reply:
Most of these I have faint but I'm not 100% on their functioning. Or why are they how they are.

E.g. Why isn't a glass drawable?